### PR TITLE
shell: periodic statitics for ping6

### PR DESCRIPTION
**Pull request by OlegHahm**
With this PR one configure via an additional argument to have periodic ping statistics after each `n` pings. This is particular helpful for long-term ping experiments.

~~Depends on #4141.~~

### CODE REVIEW

**Comment by cgundogan** [comment]
rebase please

**Comment by OlegHahm** [comment]
rebased

File: sys/shell/commands/sc_icmpv6_echo.c [hunk 1]
```diff
@@ -168,6 +197,9 @@ int _icmpv6_ping(int argc, char **argv)
     if (argc > (3 + param_offset)) {
         _a_to_timex(&delay, argv[3 + param_offset]);
     }
+    if (argc > (4 + param_offset)) {
```
Comment (cgundogan) [inline sys/shell/commands/sc_icmpv6_echo.c]: can you add this additional parameter also to the usage text that is printed when `ping6` is invoked without any arguments?

Comment (OlegHahm) [inline sys/shell/commands/sc_icmpv6_echo.c]: Ah, yes, of course. Addressed.

**Comment by cgundogan** [comment]
great, tested again and it looks great. Could you squash again please so that Travis does its duty?

**Comment by OlegHahm** [comment]
squashed

**Comment by OlegHahm** [comment]
@cgundogan, that was an ACK, right?

**Comment by cgundogan** [comment]
>@cgundogan, that was an ACK, right?

Yes. Travis agrees. GO.

**Pull request merged**